src/pages/Encounters/tabs/EncounterOverviewTab.tsx

```
@@ -1,17 +1,26 @@
+import careConfig from "@careConfig";
+import { useQuery } from "@tanstack/react-query";
+import { Plus } from "lucide-react";
+import { usePathParams } from "raviger";
+        
 import { useQuery } from "@tanstack/react-query";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useQuery'. Consider to delete it or rename it.

'useQuery' is defined here:


(lint/suspicious/noRedeclare)